lib/expectations_runner.rb

```diff
@@ -29,8 +30,9 @@ class ExpectationsRunner
   include Mumukit
   include StonesSpec::WithTempfile
 
-  def run_expectations!(expectations, content)
-    ast = generate_ast!(content)
+  def run_expectations!(expectations, content, extra = '')
+    source = TestCompiler.new.create_compilation!('examples: []', extra, content)[:source]
```

Comment:
Hmm, i would have extracted a third component here, that does just the concatenation. It could be just a mixin, and then both compiler and expectation's runner would include it and use it